docs/docs/1.4.1/deployments/how-to-multi-zone.md

```diff
@@ -0,0 +1,311 @@
+# Set up a multi-zone deployment
+
+For a description of how multi-zone deployments work in Kuma, see [about multi-zone deployments](../how-multi-zone-works). This page explains how to configure and deploy Kuma in a multi-zone environment:
+
+- Set up the global control plane
+- Set up the zone control planes
+- Verify control plane connectivity
+- Set up cross-zone communication between data plane proxies
+
+## Set up the global control plane
+
+The global control plane must run on a dedicated cluster, and cannot be assigned to a zone.
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes"
+
+The global control plane on Kubernetes must reside on its own Kubernetes cluster, to keep its CRDs separate from the CRDs the remote control planes create during synchronization.
```

Comment:
Is CRD the right word? To me a CRD is the definition of a resource, instances of CRDs are just resources. So the reason we need them separated is to keep the global resources separated from the zone resources no?